unidecode/__init__.pyi

```diff
@@ -0,0 +1,11 @@
+from typing import Any, Dict, Optional, Sequence
+
+Cache: Dict[int, Optional[Sequence[Optional[str]]]]
+
+class UnidecodeError(ValueError):
+    index: int = ...
+    def __init__(self, message: str, index: Optional[int] = ...) -> None: ...
+
+def unidecode_expect_ascii(string: str, errors: str = ..., replace_str: str = ...): ...
+def unidecode_expect_nonascii(string: str, errors: str = ..., replace_str: str = ...): ...
```

Comment:
These two should probably be `-> str`.

Reply:
Clearly! That was the whole point of typing :-)